sdk/script/management/account-manager.ts

```diff
@@ -73,7 +74,7 @@ export class AccountManager {
                 req = req.send({ identity: "accessKey" })
```

Comment:
Keys will be migrated on the server side, but on the client side, people who used the CLI prior to the change will have them persisted the old way. The format we are persisting them is different, and this code handles that.